README.md

```diff
@@ -238,17 +236,22 @@ These are not blockers for any of the core sequence and so they are worked into
 
 ### Optional / Elective Lessons
 
-- [SEO: Search Engine Optimization](/seo/seo.md)
-- [Sass](/sass/sass.md)
-- [Python](/python/python.md)
-- [Datetime](/datetime/datetime.md)
-- [Data Visualization](/data-visualization/data-visualization.md)
+- [Sass](/electives/sass.md)
+- [Python](/electives/python.md)
+- [Data Visualization](/electives/data-visualization.md)
 - [Adding Chai/Mocha Tests to your project](/electives/mocha-testing.md)
-- [MongoDB](/databases/mongo-db.md)
-- [Mongoose](/databases/mongoose.md)
-- [Auth0](/oauth/o-auth.md)
-- [Memoization](/runtime-complexity/memoization.md)
-- [Mobile Development](/mobile-development/mobile-development.md)
+- [MongoDB](/electives/databases/mongo-db.md)
+- [Mongoose](/electives/databases/mongoose.md)
+- [Auth0](/electives/oauth/o-auth.md)
+- [Memoization](/electives/memoization.md)
+- [Mobile Development](/electives/mobile-development.md)
+- [web-patterns](/electives/web-patterns.md)
+- [Database Migrations](/electives/databases/database-migrations.md)
+- [backend](/electives/1_intro_to_backend.md)
+- [Optimization](/electives/optimization.md)
+- [angular](/electives/angular.md)
+- [Asset Optimization](/electives/optimization.md)
```

Comment:
```suggestion
- [Asset Optimization](/electives/optimization.md)
- 
```
What is going on with line 251, 253, and 254? Are they the same file?
are these really going to be called /electives/optimization.md and /electives/optimization/optimization.md? Please make sure they are named correctly and there are no repeats.

Reply:
Also, where is "optimizing-your-react-node-project.md"?

Reply:
As you mentioned earlier, optimization.md is an elective. Is "optimizing-your-react-node-project.md" also an elective?

Reply:
I have just moved ../optimization.md/optimization.md to electives section.